examples/hybrid/types.jl

```diff
@@ -126,12 +129,39 @@ function get_spaces(parsed_args, params, comms_ctx)
     dz_top = FT(parsed_args["dz_top"])
     topography = parsed_args["topography"]
 
+    @assert topography in ("NoWarp", "DCMIP200", "Earth")
     if topography == "DCMIP200"
         warp_function = CA.topography_dcmip200
     elseif topography == "NoWarp"
         warp_function = nothing
+    elseif topography == "Earth"
+        data = NCDataset("/groups/esm/asridhar/ETOPO1_Ice_g_gdal.grd")
```

Comment:
This may cause problem when run it on ci or someone's desktop. Could you add the [artifacts](https://github.com/CliMA/ClimaAtmos.jl/tree/main/artifacts) download feature to access data ([e.g.](https://github.com/CliMA/ClimaAtmos.jl/blob/main/test/non_orographic_gravity_wave/nogw_test_3d.jl#L54-L67))?